src/libmongoc/tests/test-mongoc-sample-commands.c

```diff
@@ -2676,6 +2675,157 @@ test_example_59 (mongoc_database_t *db)
    /* End Snapshot Query Example 1 Post */
 }
 
+static bool
+retail_setup (mongoc_collection_t *sales_collection)
+{
+   bool ok = true;
+   bson_t *doc = NULL;
+   bson_error_t error;
+   struct timeval tv;
+   int64_t unix_time_now = 0;
+
+   if (bson_gettimeofday (&tv)) {
+      MONGOC_ERROR ("could not get time of day");
+      goto cleanup;
+   }
+   unix_time_now = 1000 * tv.tv_sec;
+
+   mongoc_collection_drop (sales_collection, NULL);
+
+   doc = BCON_NEW ("shoeType",
+                   BCON_UTF8 ("boot"),
+                   "price",
+                   BCON_INT64 (30),
+                   "saleDate",
+                   BCON_DATE_TIME (unix_time_now));
+
+   ok =
+      mongoc_collection_insert_one (sales_collection, doc, NULL, NULL, &error);
+   if (!ok) {
+      MONGOC_ERROR ("insert into retail.sales failed: %s", error.message);
+      goto cleanup;
+   }
+
+cleanup:
+   bson_destroy (doc);
+   return ok;
+}
+
+
+static void
+test_example_60 (mongoc_database_t *db)
+{
+   /* Start Snapshot Query Example 2 */
+   mongoc_client_t *client = NULL;
+   mongoc_client_session_t *cs = NULL;
+   mongoc_collection_t *sales_collection = NULL;
+   bson_error_t error;
+   mongoc_session_opt_t *session_opts;
+   bson_t *pipeline = NULL;
+   bson_t opts = BSON_INITIALIZER;
+   mongoc_cursor_t *cursor = NULL;
+   const bson_t *doc = NULL;
+   bool ok = true;
+   bson_iter_t iter;
+   int64_t total_sales = 0;
+
+   client = test_framework_new_default_client ();
```

Comment:
I suggest constructing `client` outside of the example comments. `test_framework_new_default_client` is not a public API function.

I missed this before. Please also update test_example_59.